downloader/src/main/kotlin/VersionControlSystem.kt

```diff
@@ -82,6 +82,12 @@ abstract class VersionControlSystem {
                     }
                 }
 
+        /**
+         * Return the relative path to [file] with respect to the VCS root or null if [file] is not in a VCS repository.
+         */
+        fun pathToRoot(file: File) = VersionControlSystem.forDirectory(file.takeIf { it.isDirectory }
```

Comment:
The `VersionControlSystem.` prefix can be dropped here.

Should we call this also `getPathToRoot()` and explicitly add to the docs that this is a convenience wrapper around `WorkingTree.getPathToRoot()` that creates a temporary working tree based on `file`?

Reply:
Done.